dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/audit/AuditReplayReducer.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.dynamometer.workloadgenerator.audit;
+
+import org.apache.hadoop.io.LongWritable;
+import org.apache.hadoop.mapreduce.Reducer;
+
+import java.io.IOException;
+
+public class AuditReplayReducer extends
+    Reducer<UserCommandKey, LongWritable, UserCommandKey, LongWritable> {
+
+  @Override
+  protected void reduce(UserCommandKey key, Iterable<LongWritable> values, Context context) throws IOException, InterruptedException {
+    long sum = 0;
```

Comment:
If you want: `long sum = StreamSupport.stream(values.spliterator(), false).mapToLong(LongWritable::get).sum()`

Reply:
Not convinced that this is better 🤔